webpack.config.js

```diff
@@ -39,8 +39,8 @@ const extractConfig = {
 const entryPointNames = [
 	'element',
 	'i18n',
-	'components',
 	'utils',
+	'components',
```

Comment:
`blocks` now depends on `utils`, so `utils` should be loaded first.  Output from the script at #965:

```
'blocks' depends on 'components'
'blocks' depends on 'element'
'blocks' depends on 'i18n'
'blocks' depends on 'utils'
'components' depends on 'element'
'components' depends on 'i18n'
'components' depends on 'utils'
'editor' depends on 'blocks'
'editor' depends on 'components'
'editor' depends on 'date'
'editor' depends on 'element'
'editor' depends on 'i18n'
'editor' depends on 'utils'

Suggested load order:

element
i18n
utils
components
blocks
date
editor
```